dashboard/app/controllers/aichat_controller.rb

```diff
@@ -160,6 +160,11 @@ def check_message_safety
     render(status: :ok, json: response_body)
   end
 
+  # GET /aichat/user_has_aichat_access
```

Comment:
nit: maybe `/aichat/has_access` for simplicity since we already have `aichat` in the path?

Reply:
Sure. I updated to `user_has_access` because `has_access` resulted in a lint error asking for `?` at the end.